### ISSUE TITLE
Publish new release

### ISSUE THREAD

**Issue by jtpio** [jupyterlite/jupyterlite-sphinx#171]
Opening a new issue to discuss publishing a new release of `jupyterlite-sphinx`.

https://github.com/jupyterlite/jupyterlite-sphinx/pull/160 relaxed the dependency on `jupyterlite-core`, so `jupyterlite-sphinx` can be used with the latest JupyterLite 0.3.0 release.

This would also help unblock https://github.com/jupyter/try-jupyter/pull/49.

**Comment by jtpio** [comment]
cc @steppi @Carreau @martinRenou, just in case there were more PRs you wanted to get merged first before cutting a new release

**Comment by agriyakhetarpal** [comment]
I have a stash locally where I was recently trying to disable the "Try it!" buttons by default, and selectively enable them based on the JSON configuration (discussed with @steppi). I wasn't able to get it fully working back then, but I'm happy to retry that before publishing a new release if it helps.

**Comment by steppi** [comment]
@jtpio, no holdups on my end. I'd like to see the stuff @agriyakhetarpal is working on get in, but I don't think it's worth holding up a release for.

**Comment by jtpio** [comment]
Nice. Should we go with `0.14.0`? Not sure if the other changes that were merged could go into a `0.13.x` release.
